lib/handlers/subServiceRequested/subServiceRequestToApprovers.js

```diff
@@ -0,0 +1,68 @@
+const { directories } = require('login.dfe.dao');
+const { getEmailAdapter } = require('../../infrastructure/email');
+const OrganisatonsClient = require('../../infrastructure/organisations');
+const DirectoriesClient = require('../../infrastructure/directories');
+
+const jobType = 'sub_service_request_to_approvers';
+
+const execute = async (config, logger, data) => {
+  try {
+    const organisationsClient = new OrganisatonsClient(config.notifications.organisations);
+    const directoriesClient = new DirectoriesClient(config.notifications.directories);
+
+    const isMultipleSubServicesReq = data.requestedSubServices.length > 1;
+    let subject = isMultipleSubServicesReq
+      ? 'A user has requested access to multiple sub-services'
+      : 'A user has requested access to a sub-service';
+
+    const { envName } = config.notifications;
+    if (envName !== 'pr') {
```

Comment:
what is envName !== 'pr'? do we need a default value?

Reply:
'pr' is the env name we had in config files for Prod - this condition will add the env name in the email subject in the lower envs to differentiate lower envs requests from PROD requirements - no default value is needed